app/views/transactions.html

```diff
@@ -88,27 +98,34 @@ <h3 style="margin: 20px;" ng-if="transactionsSelected.length < defaultBulkRerunL
                 </tr>
               </thead>
               <tbody>
-                <tr class="table-list" ng-repeat="transaction in transactions" ng-click='viewTransactionDetails("transactions/" + transaction._id, $event)'>
+                <tr class="table-list" ng-repeat="transaction in transactions" ng-style="baseIndex + $index < 0 && {'color': '#036633', 'font-weight': 'bold'}" ng-click='viewTransactionDetails("transactions/" + transaction._id, $event)'>
```

Comment:
Maybe rather apply a css class rather than applying the css inline. I'm not quite sure what this is doing here either, mind explaining a bit.